doc/Protocols/pr-mserv_x-road_protocol_for_management_services.md

```diff
@@ -313,7 +326,62 @@ The request is sent using HTTP POST method. The content type of the request MUST
 
 The response echoes back the server and the client fields of the request and adds the field *requestId*.
 
-An example of the address change request and response is given in [Annex A.6](#a6-addressChange)).
+An example of the address change request and response is given in [Annex A.6](#a6-addresschange)).
+
+### 2.7 *clientDisable* - Disable Security Server Client Subsystem Temporarily
+
+The *clientDisable* service is used to disable Security Server client subsystem temporarily.
+
+The body of the client disabling message (request or response) contains following fields:
+
+* **client** – identifier of the subsystem to be disabled;
+* **server** – identifier of the Security Server where the client is disabled;
+* **requestId** – for responses only, unique identifier of the request that is stored in the Central Server database \[[DM-CS](#Ref_DM-CS)\].
+
+The XML Schema fragment of the client deletion request body shown below.
+
+```xml
+<xsd:complexType name="ClientRequestType">
+    <xsd:sequence>
+        <xsd:element name="server" type="id:XRoadSecurityServerIdentifierType"/>
+        <xsd:element name="client" type="id:XRoadClientIdentifierType"/>
+        <element name="requestId" type="tns:RequestIdType" minOccurs="0"/>
+    </xsd:sequence>
+</xsd:complexType>
+```
+
```

Comment:
It's the Security Server owner's signature as with other types of requests.

Reply:
Actually, it's not always the owner member's signature:

- clientReg - the member that owns the subsystem to be registered as a Security Server client;
- authCertReg - the Security Server's owner;
- ownerChange - the new owner member of the Security Server;
- addressChange - the Security Server's owner.

In this case, a better alternative might be the member that owns the subsystem to be enabled/disabled. In that way, the implementation would be aligned with the `clientReg` management service. Since registering a subsystem is signed by the member owning the subsystem, it would make sense that also enabling / disabling the subsystem would be signed by the subsystem owner.